how2meet/ui/pages/events.py

```diff
@@ -110,30 +99,35 @@ async def delete(invite_uuid=invite_uuid):
             with ui.column().classes("w-full"):
                 add_guest_button = ui.button("Add guest", on_click=add_guest)
 
-        async def get_event_json():
+        async def get_event_json() -> str:
```

Comment:
If this function isn't returning an actual JSON type, should it be named differently?

Reply:
@adrastaea wrote this, and I have questions about it too.  I was fucking with stuff and the routes are _supposed_ to accept `dict-like` objects in the `data=` field, but they don't work if you do. It has to be a JSON formatted string (or so it seems).